apps/token-manager/app/src/screens/Holders.js

```diff
@@ -1,166 +1,163 @@
-import React from 'react'
+import React, { useCallback, useState } from 'react'
 import PropTypes from 'prop-types'
+import { useNetwork } from '@aragon/api-react'
 import styled from 'styled-components'
 import {
+  ContextMenu,
+  ContextMenuItem,
+  DataView,
+  IconAdd,
+  IconLabel,
+  IconRemove,
+  Split,
   TabBar,
   Table,
   TableHeader,
   TableRow,
   Viewport,
   breakpoint,
+  useTheme,
 } from '@aragon/ui'
-import HolderRow from '../components/HolderRow'
-import SideBar from '../components/SideBar'
+import { formatBalance } from '../utils'
+import InfoBoxes from '../components/InfoBoxes'
+import LocalIdentityBadge from '../components/LocalIdentityBadge/LocalIdentityBadge'
+import { useIdentity } from '../components/IdentityManager/IdentityManager'
 
-const TABS = ['Holders', 'Token Info']
+function Holders({
+  groupMode,
+  holders,
+  maxAccountTokens,
+  onAssignTokens,
+  onRemoveTokens,
+  tokenAddress,
+  tokenDecimalsBase,
+  tokenName,
+  tokenSupply,
+  tokenSymbol,
+  tokenTransfersEnabled,
+  userAccount,
+}) {
+  const network = useNetwork()
+  return (
+    <Split
+      primary={
+        <DataView
+          mode="table"
+          fields={groupMode ? ['Owner'] : ['Holder', 'Balance']}
+          entries={holders.map(({ address, balance }) =>
+            groupMode ? [address] : [address, balance]
+          )}
+          renderEntry={([address, balance]) => {
+            const isCurrentUser = Boolean(
+              userAccount && userAccount === address
+            )
 
-class Holders extends React.Component {
-  static propTypes = {
-    holders: PropTypes.array,
-  }
-  static defaultProps = {
-    holders: [],
-  }
-  state = { selectedTab: 0 }
-  render() {
-    const {
-      groupMode,
-      holders,
-      maxAccountTokens,
-      onAssignTokens,
-      onRemoveTokens,
-      tokenAddress,
-      tokenDecimalsBase,
-      tokenName,
-      tokenSupply,
-      tokenSymbol,
-      tokenTransfersEnabled,
-      userAccount,
-    } = this.props
-    const { selectedTab } = this.state
+            const values = [
+              <React.Fragment>
+                <LocalIdentityBadge
+                  entity={address}
+                  networkType={network.type}
+                  connectedAccount={isCurrentUser}
+                />
+                {isCurrentUser && <You />}
+              </React.Fragment>,
+            ]
 
-    return (
-      <Viewport>
-        {({ below }) => {
-          const tabbedNavigation = below('medium')
-          const compactTable = below('medium')
+            if (balance) {
```

Comment:
I see you used the same pattern here, in case my other comment resonated and you updated I'd suggest you do the same here, if not, then nothing to see here.